### ISSUE TITLE
This appears to still have pr2 specific messages that are unrelated to computers

### ISSUE THREAD

**Issue by tfoote** [plusone-robotics/computer_status_msgs#2]
Reading through the messages several appear to be unrelated to computers and are PR2 specific thus should probably not be here: 

- PowerBoardState PR2 specific
- AccelerometerState  not computer related, message has lots of PR2 content in it
- BatteryServer  -- pretty PR2 specific
- BatteryState -- implmentation is pretty PR2 specific with undefined registers in the message. Also redundant with [sensor_msgs/BatteryState](http://docs.ros.org/api/sensor_msgs/html/msg/BatteryState.html) 
- Dashboard State -- completely PR2 specific
- PeriodicCmd -- completely PR2 specific
- PressureState -- completely PR2 specific
- PowerState -- Seemingly pretty redundant with BatteryState it was used as an aggregate of the PR2 specific battery states. 

This was [migrated from pr2_msgs](https://github.com/PR2/pr2_common/issues/286) but I think there should have been a lot more pruning instead of bulk copying.

**Comment by v4hn** [comment]
I agree. I was giving a number of examples in [my post in that issue](https://github.com/PR2/pr2_common/issues/286#issuecomment-695975839), but it looks like Isaac relied on this as an exhaustive list.

**Comment by 130s** [comment]
@tfoote @v4hn Thank you for the feedback!

- For "PR2 specific" stuff, I'd like to argue once. In the message format that I moved to this package, I'm having a trouble to find why you call them as PR2 specific. If a user see those msg definition without knowing the background of these msgs, they might think these msgs are useful for their non-PR2 robots (like I do now).
- Would it make sense to send some of these msgs to `sensor_msgs`? I'm happy to do the maintenance on this repo but if it can be added to `sensor_msgs` that's far better because it gets a lot more visibility (hence better maintenance opportunity). I understand the bar to add a new msg at `sensor_msgs` is high, but a good thing about PR2 msgs is they're probably used for the longest time in ROS history. I'm particularly in need of `GPUStatus.msg` for now. 
- For "not computer related" things and a redundant msg, I agree to remove from "computer_status_msgs".

(@v4hn I took your advice in https://github.com/plusone-robotics/computer_status_msgs/pull/1 btw, not fully though).
